File: 1010.js
/*
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};
*/

var [A, B, C] = prompt("").split(" ");
parseInt(A);
parseInt(B);
parseFloat(C);
var [D, E, F] = prompt("").split(" ");
parseInt(D);
parseInt(E);
parseFloat(F);
var res = (B * C) + (E * F);
console.log(`VALOR A PAGAR: R$ ${res.toFixed(2)}`);

File: Python/1010.py
total_a_pagar = 0
for _ in range(2):
    valores = input().split()
    codpec = int(valores[0])
    numpec = int(valores[1])
    valpec = float(valores[2])
    total_a_pagar += numpec * valpec
print('VALOR A PAGAR: R$ %.2f' % total_a_pagar)